src/payment/models/bank-tx/olkypay.service.ts

```diff
@@ -88,17 +88,20 @@ export class OlkypayService {
   }
 
   private async getTransactions(fromDate: Date, toDate: Date = new Date()): Promise<Transaction[]> {
-    const url = `ecritures/${Config.bank.olkypay.clientId}/${Util.isoDate(fromDate)}/${Util.isoDate(toDate)}`;
+    const url = `ecritures/${Config.bank.olkypay.clientId}/${Util.isoDate(fromDate)}/${Util.isoDate(
+      Util.daysAfter(1, toDate),
```

Comment:
I would move the daysAfter to the calling method, because it IMO disturbs the logic of this method (if we one day e.g. want to get all transactions of a period in the past).